pheweb/serve/server_jeeves.py

```diff
@@ -424,13 +424,44 @@ def get_autoreport(self, phenocode):
         return []
         
     
-    def get_autoreport_variants(self, phenocode, locus_id):
-        if self.autoreporting_dao is None:
+    def get_autoreport_variants(self, phenocode: str, locus_id: str) -> List[Dict[str,Union[str,int,float,bool]]]:
```

Comment:
Don't need to do now but maybe when moving to the new more modular framework.
Good old story of moving away from these cryptic dictionaries and moving to proper classes and maybe putting them to the common data mode. Goes to all methods of auterepdb

Reply:
Then there is no reason to cast to pandas object and then use that to loop etc. etc.